app/models/ZkKafka.scala

```diff
@@ -40,20 +40,26 @@ object ZkKafka {
   def getTopologies: Seq[Topology] = {
     zkClient.getChildren.forPath(makePath(Seq(stormZkRoot))).asScala.map({ r =>
       getSpoutTopology(r)
-    }).sortWith(topoCompFn)
+    }).flatten.sortWith(topoCompFn)
```

Comment:
To clarify, this modifies `getTopologies` to first yield a `List[Option[Topology]]` and then `flatten` it by plucking the `Topology` out of the `Option[Topology]`.  This certainly fixes the problem, but it doesn't notify the user in any way that we had to throw away a value. In addition to the above perhaps modify the function inside the `map` of `getChildren` to check the definedness of the returned `Option` and log the `r` value that had problems?